src/netlink/nl_l3.cc

```diff
@@ -356,6 +351,17 @@ int nl_l3::add_l3_addr_v6(struct rtnl_addr *a) {
     }
   }
 
+  if (auto members = nl->get_bond_members_by_lag(link); !members.empty()) {
+    VLOG(1) << __FUNCTION__ << ": BOND configuring slaves";
+
+    for (auto mem : members) {
+      auto _link = nl->get_link_by_ifindex(nl->get_ifindex_by_port_id(mem));
+      bool tagged = !!rtnl_link_is_vlan(link);
+
+      rv = vlan->add_vlan(_link.get(), vid, tagged, 0);
```

Comment:
Actually 0 is the default passed when omitting the argument, so you can just drop it
```suggestion
      rv = vlan->add_vlan(_link.get(), vid, tagged);
```